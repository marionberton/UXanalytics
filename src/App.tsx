import React from 'react';

// import ReactGA from 'react-ga';

import { library } from '@fortawesome/fontawesome-svg-core';
import { fab } from '@fortawesome/free-brands-svg-icons';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';

import { Navbar } from './Landing/components/shared/navbar';
import { Main } from './Landing/pages/main';
import { Footer } from './Landing/components/shared/footer';
import { Auth } from './Auth/Auth';

import './styles/tailwind.css';

library.add(fab);

// ReactGA.initialize('UA-00000-1');
export const App = () => {
  return (
    <>
      <Router>
        <Switch>
          <Route path="/" exact>
            <div className="backgroundn">
              <Navbar />
              <Main />
              <Footer />
            </div>
          </Route>
          <Route path="/signin" exact>
            <Auth />
          </Route>
        </Switch>
      </Router>
    </>
  );
};
