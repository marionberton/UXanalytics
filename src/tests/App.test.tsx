import React from 'react';
import { shallow } from 'enzyme';

import { App } from '../App';

describe('<Root />', () => {
  it('renders the component', () => {
    const component = shallow(<App />);

    expect(component).toMatchSnapshot();
  });
});
