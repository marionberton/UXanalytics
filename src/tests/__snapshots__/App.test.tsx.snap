// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Root /> renders the component 1`] = `
<Fragment>
  <BrowserRouter>
    <Switch>
      <Route
        exact={true}
        path="/"
      >
        <div
          className="backgroundn"
        >
          <Component />
          <Component />
          <Component />
        </div>
      </Route>
      <Route
        exact={true}
        path="/signin"
      >
        <Component />
      </Route>
    </Switch>
  </BrowserRouter>
</Fragment>
`;
